apps/notifications/src/panel/state/index.js

```diff
@@ -13,13 +14,14 @@ const reducer = combineReducers( {
 
 /** @see https://github.com/zalmoxisus/redux-devtools-extension */
 const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+
+const listenerMiddleware = createListenerMiddleware();
 const withMiddleware = ( customMiddleware ) =>
-	composeEnhancers( applyMiddleware( thunkMiddleware, actionMiddleware( customMiddleware ) ) )(
-		createStore
-	);
+	composeEnhancers(
+		applyMiddleware( thunkMiddleware, listenerMiddleware, actionMiddleware( customMiddleware ) )
+	)( createStore );
```

Comment:
The `customMiddleware` parameter of `init` is no longer used, it's been replaced by adding listeners. So it can be removed. Also the `actionMiddleware` can be completely removed.

Reply:
The `actionMiddleware` still has some default middlewares (handlers) set, so we still need it. We can migrate them to `listenerMiddleware` but I'd rather keep this PR small. We can certainly do that in another PR though!

Reply:
You're right, `actionMiddleware` still needs to be registered, we just can remove the `customMiddleware` parameter, and register it with a simple `actionMiddleware()` call.

Can be done in another PR so that this one doesn't get too complicated.